spacewalk/certs-tools/mgr-package-rpm-certificate-osimage

```diff
@@ -47,8 +49,12 @@ def processCommandline():
 
 def genCaRpm(options):
     OSIMAGE_RPM_CERTIFICATE_PATH = "/usr/share/susemanager/salt/images"
-    CA_CERT_RPM_NAME_OSIMAGE = CA_CRT_RPM_NAME + "-osimage"
-    OSIMAGE_RPM_REQUIRES = "ca-certificates"
+    if options.target_os == 'SLE11':
+        CA_CERT_RPM_NAME_OSIMAGE = CA_CRT_RPM_NAME + "-osimage-sle11"
+        OSIMAGE_RPM_REQUIRES = ["openssl-certs", "coreutils"]
```

Comment:
`openssl-certs` is the equivalent in SLES11 for `ca-certificates`?

Reply:
The post-install script on SLES11 calls binaries from openssl-certs and coreutils to deploy the certificate:
https://github.com/uyuni-project/uyuni/blob/master/spacewalk/certs-tools/update-ca-cert-trust.sh